libnavigation-core/api/current.txt

```diff
@@ -32,6 +32,7 @@ package com.mapbox.navigation.core {
     method public void resetTripSession();
     method public String retrieveHistory();
     method public String? retrieveSsmlAnnouncementInstruction(int index);
+    method public com.mapbox.navigation.core.MapboxNavigation routeRefreshInterval(long routeRefreshInterval);
```

Comment:
I'd personally prefer option `#2`, the `Options` pattern, for consistency. We haven't heard feature requests for this API to be mutable in the runtime, so I think we should make the functional surface as small as possible.

Not a blocker, but something to consider.

Reply:
Thanks for extra opinion here. Let's go with consistency first. When/if there are developer requests to have dynamic settings (example: change the interval while in route), we will also be able to evaluate solutions that work with other features too